### PR TITLE
Update Flake8 settings descriptions

### PR DESCRIPTION
**Pull request by luabud**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -221,4 +221,4 @@
         "webpack": "^5.89.0",
         "webpack-cli": "^5.1.4"
     }
-}
+}
```
Comment (karthiknadig) [inline package.json:224]:
```suggestion
}

```

File: package.nls.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+    "settings.importStrategy.fromEnvironment.description": "Use Flake8 from the selected environment. If the extension fails to find a valid Flake8 binary, it will fallback to using the bundled version of Flake8.",
+    "settings.interpreter.description": "Path to a Python executable or a command that will be used to launch the Flake8 server and any subprocess. Accepts an array of a single or multiple strings.  When set to `[]`, the extension will use the path to the selected Python interpreter. If passing a command, each argument should be provided as a separate string in the array.",
+    "settings.showNotifications.description": "Controls when notifications are shown by this extension.  Accepted values are `onError`, `onWarning`, `always` and `off`.",
+    "settings.showNotifications.off.description": "Never display a notification. Any errors or warning are still available in the logs.",
+    "settings.showNotifications.onError.description": "Show notifications for errors.",
+    "settings.showNotifications.onWarning.description": "Show notifications for errors and warnings.",
+    "settings.showNotifications.always.description": "Show all notifications."
+}
```
Comment (karthiknadig) [inline package.nls.json:18]:
```suggestion
}

```

File: README.md [hunk 3]
```diff
@@ -1,34 +1,121 @@
...
 
-Note:
+> **Note**: The minimum version of Flake8 this extension supports is 5.0.0.
 
--   This extension is supported for all [actively supported versions](https://devguide.python.org/#status-of-python-branches) of the `python` language (i.e., python >= 3.8).
+This extension supports all [actively supported versions](https://devguide.python.org/#status-of-python-branches) of the Python language (i.e., Python >= 3.8).
+
+For more information on Flake8, see https://flake8.pycqa.org/
+
+-   This extension is supported for all  of the `python` language (i.e., python >= 3.8).
```
Comment (cwebster-99) [inline README.md:11]: I don't think this line is needed as the information is covered on line 7. 
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,34 +1,121 @@
...
+-   **Mono repo support**: If you are working with a mono repo, you can configure the extension to lint Python files in subfolders of the workspace root folder by setting the `flake8.cwd` setting to `${fileDirname}`. You can also set it to ignore/skip linting for certain files or folder paths by specifying a glob pattern to the `flake8.ignorePatterns` setting.
+-   **Customizable Linting Rules**: You can customize the severity of specific Flake8 error codes through the `flake8.severity` setting.
+
+### Disabling Flake8
 
-Once installed in Visual Studio Code, flake8 will be automatically executed when you open a Python file.
+You can skip linting with Flake8 for specific files or directories by setting the `flake8.ignorePatterns` setting.
 
-If you want to disable flake8, you can [disable this extension](https://code.visualstudio.com/docs/editor/extension-marketplace#_disable-an-extension) per workspace in Visual Studio Code.
+But if you wish to disable linting with Flake8 for your entire workspace or globally, you can [disable this extension](https://code.visualstudio.com/docs/editor/extension-marketplace#_disable-an-extension) in Visual Studio Code.
```
Comment (cwebster-99) [inline README.md:27]:
```suggestion
If you wish to disable linting with Flake8 for your entire workspace or globally, you can [disable this extension](https://code.visualstudio.com/docs/editor/extension-marketplace#_disable-an-extension) in Visual Studio Code.
```

File: README.md [hunk 5]
```diff
@@ -1,34 +1,121 @@
...
+
+For more information on Flake8, see https://flake8.pycqa.org/
+
+-   This extension is supported for all  of the `python` language (i.e., python >= 3.8).
 -   Minimum supported version of `flake8` is `5.0.0`.
 
-## Usage
+## Usage and Features 
+
+The Flake8 extension provides a series of features to help your productivity while working with Python code in Visual Studio Code. Check out the [Settings section](#settings) below for more details on how to customize the extension.
```
Comment (cwebster-99) [inline README.md:16]:
```suggestion
The Flake8 extension provides features to improve your productivity while working on Python code in Visual Studio Code. Check out the [Settings section](#settings) below for more details on how to customize the extension.
```

**Review by karthiknadig — APPROVED** [review]

**Review by cwebster-99 — APPROVED** [review]
LGTM

**Pull request merged**